docs/catalog/getting_started/setup.md

```diff
@@ -0,0 +1,153 @@
+These docs are part of the larger [@nulogy/design-system](https://github.com/nulogy/design-system) project. This project uses [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage multiple packages in one repository.
+
+To work on Nulogy Design System, you will need to clone the repo to your machine, install necessary dependencies and run the development server. This document will guide you through that process.
+
+# Clone the repo to your local machine
+
+The repo is located at [https://github.com/nulogy/design-system](https://github.com/nulogy/design-system). 
+
+You can clone the repo from the command line like so:
+
+```code
+lang: sh
+---
+$ cd ~/src/nulogy # or wherever you want to work
+$ git clone https://github.com/nulogy/design-system.git design-system
+$ cd design-system
+```
+
+# Installation
+
+## Yarn
+
+Nulogy Design System uses the [Yarn package](https://yarnpkg.com/) manager to manage [npm](https://www.npmjs.com/) dependencies. 
+
+You can check if you have yarn installed by typing `yarn -v` in the terminal: 
+
+
+```code
+lang: sh
+---
+$ yarn -v
+1.7.0 
+```
+
+If you see a version number like this you're good 👍
+
+However, If you see this... 
+
+```code
+lang: sh
+---
+$ yarn -v
+bash: command not found: yarn 
+```
+
+ ... then you'll need to install yarn by following the instructions for your platform on the yarn website: [https://yarnpkg.com/lang/en/docs/install](https://yarnpkg.com/lang/en/docs/install)
+
+## Install dependencies and link local packages
+
+Yarn workspaces allows you to install dependencies for all packages in the monorepo in one command.
```

Comment:
typo, extra s. "Yarn workspaces allow you..."

Reply:
I think in this case "yarn workspaces" is a collective noun and should be treated as singular.

Reply:
My editor wife suggests capitalizing the W

Reply:
Looks like Yarn can't even decide if its singular or plural:

> Workspaces **are** a new way to setup your package architecture that’s available by default starting from Yarn 1.0. **It allows** you to setup multiple packages in such a way that you only need to run yarn install once to install all of them in a single pass.

Reply:
I rewrote the sentence to add a more stable subject